docs/design/autoware-interfaces/components/planning.md

```diff
@@ -1,5 +1,31 @@
 # Planning
 
+| Input From Perception Component | Topic Name(Data Type)                                                                                               | Explanation                                                                                                                                                                                          |
+| ------------------------------- | ------------------------------------------------------------------------------------------------------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Detected Object Information     | `/perception/object_recognition/objects`<br>(_autoware_auto_perception_msgs/msg/PredictedObjects_)                  | This includes information that cannot be known beforehand such as pedestrians and other vehicles. Planning stack will plan maneuvers to avoid collision with such objects.                           |
+| Detected Obstacle Information   | `/perception/obstacle_segmentation/pointcloud`<br>(_sensor_msgs/msg/PointCloud2_)                                   | This includes information on the location of obstacles. This is more primitive information and is used for emergency stops, etc.                                                                     |
+| Occupancy Map Information       | `/perception/occupancy_grid_map/map`<br>(_nav_msgs/msg/OccupancyGrid_)                                              | This includes information that cannot be known beforehand such as pedestrians and other vehicles. Planning stack will plan maneuvers to avoid collision with such objects.                           |
+| TrafficLight recognition result | `/perception/traffic_light_recognition/traffic_signals`<br>(_autoware_auto_perception_msgs/msg/TrafficSignalArray_) | This is the real time information about the state of each traffic light. Planning stack will extract the one that is relevant to planned path and use it to decide whether to stop at intersections. |
+
+| Input From Localization Component | Topic Name(Data Type)                                        | Explanation                                                |
+| --------------------------------- | ------------------------------------------------------------ | ---------------------------------------------------------- |
+| Vehicle Kinematics                | `/localization/kinematic_state`<br>(_nav_msgs/msg/Odometry_) | This includes ego vehicle's pose and velocity information. |
+| Vehicle Acceleration              | `/localization/acceleration`<br>(\_)                         | This includes ego vehicle's acceleration.                  |
```

Comment:
the data type is missing.

Reply:
autoware-interface docs are not updated yet. It is removed from this PR.